.github/workflows/ci-test.yml

```diff
@@ -533,7 +533,7 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: unit-test-results
-          path: target/junit-reports/unit-test.xml
+          path: target/nextest-reports/nextest-junit.xml
```

Comment:
This seems fine zip the shell to extract these reports will always retain the artifact name.   Other use cases may not be so lucky.

Reply:
Yeah I was relying on that. I think it's fine since a report still is uniquely identified by the (workflow run ID, artifact name, file name) triple.